plutus-ledger-constraints/src/Ledger/Constraints/OffChain.hs

```diff
@@ -16,6 +16,7 @@
 {-# LANGUAGE TypeOperators             #-}
 {-# LANGUAGE UndecidableInstances      #-}
 {-# LANGUAGE ViewPatterns              #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
```

Comment:
Why was this added? Can't we just remove the unused imports?

Reply:
Fixed by 'Fix imports.'